Source/Renderer/RiveRenderer.mm

```diff
@@ -388,7 +388,8 @@
         }
         
         // Clip the gradient
-        CGContextClip(ctx);
+        if (!CGContextIsPathEmpty(ctx))
```

Comment:
Nice

Reply:
Though not sure why we're getting empty paths in the first place. Are be parting over something?